core/src/main/java/hudson/node_monitors/DiskSpaceMonitor.java

```diff
@@ -61,7 +63,17 @@ public String getColumnCaption() {
         return Jenkins.get().hasPermission(Jenkins.ADMINISTER) ? super.getColumnCaption() : null;
     }
 
-    public static final DiskSpaceMonitorDescriptor DESCRIPTOR = new DiskSpaceMonitorDescriptor() {
+    @SuppressFBWarnings(value = "MS_PKGPROTECT", justification = "for backward compatibility")
+    public static /*almost final*/ DiskSpaceMonitorDescriptor DESCRIPTOR;
+
+    @Extension @Symbol("diskSpace")
```

Comment:
Looks risky given line 95. Does this create a duplicate ExtensionList entry? Is this newly invocation order dependent?

Reply:
ahh I missed that line 95. Fixed it hopefully.